app/views/flood_risk_engine/shared/_manual_address.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="govuk-form-group <%= "govuk-form-group--error" if form.errors[:premises].any? %>">
+  <% if form.errors[:premises].any? %>
+    <span class="govuk-error-message"><%= form.errors[:premises].join(", ") %></span>
+  <% end %>
+
+  <%= f.govuk_text_field :premises,
+      label: { text: t(".premises_label") },
+      hint: { text: t(".premises_hint") },
+      width: "one-half",
+      value: form.premises
+  %>
+</div>
+
+<div class="govuk-form-group <%= "govuk-form-group--error" if form.errors[:street_address].any? %>">
```

Comment:
Do we have access to `form` here? It might just be `f`. The errors should show up with the `govuk_text_field` helper, so the form-group wrapper may not be needed. Might be worth having another look

Reply:
In this case `form` is representing the form instance variable, eg `@partner_manual_address_form`, so that's why it's different to `f`. There aren't really any other good examples of it in FRAE but we have similar things in WCR, for example: https://github.com/DEFRA/waste-carriers-engine/blob/main/app/views/waste_carriers_engine/shared/_dob.html.erb